src/app/modules/consultations/consultation-profile/read-respond/read-respond.component.ts

```diff
@@ -186,6 +209,25 @@ export class ReadRespondComponent implements OnInit, AfterViewChecked {
     }
   }
 
+  onAnswerChange(question?, value?) {
+    if (question && value.id === 'other') {
+      for (let i = 0; i < this.profileData.questions.length; i++) {
+        if (this.profileData.questions[i].id === question.id) {
+          this.profileData.questions[i].is_other = true;
+          break;
+        }
+      }
+    } else {
```

Comment:
@preethijayanthi What is this block for?

Reply:
When I click other option and click some other option to hide the input box.

Reply:
@preethijayanthi I think one more improvement can be made here. Right now in the case of checkbox type questions, even if the other option is unchecked, the input box still there. If others option is unchecked in a checkbox type question, the others input box should also be hidden